server/src/server.js

```diff
@@ -0,0 +1,156 @@
+const { v4: uuidv4 } = require("uuid");
+const express = require("express");
+const cookieSession = require("cookie-session");
+const datastore = require("./datastore");
+const app = express();
+
+const KIND_EVENT = "Event";
+const URL_PARAM_EVENT_ID = `eventID`;
+
+const ERROR_BAD_DB_INTERACTION = "BAD_DATABASE";
+const ERROR_INVALID_EVENT_ID = "INVALID_EVENT_ID";
+const ERROR_BAD_UUID = "BAD_UUID";
+
+// Parse request bodies with the json content header into JSON
+app.use(express.json());
+
+app.use(
+  cookieSession({
+    name: "session",
+    keys: ["secret1"],
+    maxAge: 365 * 24 * 60 * 60 * 1000, // 1 year
+  })
+);
+
+app.use((req, res, next) => {
```

Comment:
prefer to avoid partial names - this is generally a pointless holdover from the days of C when length limitations were common. abbreviations and shortened names may be acceptable in some cases where they are extremely widely known and understood (i instead index, req instead of request for instance), but generally you should default to avoiding them unless there are extenuating circumstances. i would argue this is doubly true in untyped languages, where it is even more critical to convey information about the variable through it's naming. ie, use request instead of req here, res instead of response (res is often used for resource as well, making this confusing).

Reply:
I generally agree. 

I chose to use `req`, `res`, `next` here because they are generally ExpressJS conventions: https://expressjs.com/en/5x/api.html#router as seen in the docs. But I will change `req` to `request` and `res` to `response`. Longer variable names generally don't hurt :)